packages/discovery-react-components/src/components/CIDocument/components/CIDocument/messages.ts

```diff
@@ -16,6 +17,7 @@ export interface Messages {
 
 export const defaultMessages: Messages = {
   defaultDocumentName: 'Document',
+  defaultDocumentId: 'Document Id',
```

Comment:
Is this a valid document id? What happens if this is left as the default?

Reply:
>Is this a valid document id?

Not sure, but it only gets passed into the key value of the virtual scroll elements.

>What happens if this is left as the default?

Then this bug can still occur when the user is switching documents. But that would happen with any default value or just leaving it as `''`. Maybe it's better to just not have this and set the default inside the component. I was just following the existing pattern when I set this up, and didn't give it incredibly close consideration.